pages/translations/en/LC_MESSAGES/messages.po

```diff
@@ -1147,7 +1150,1930 @@ msgstr ""
+msgstr ""
+
+msgid "Some things can be made fast and are still a terrible user experience. AMPs should deliver a fantastic user experience and speed is just one part of that. Only compromise when lack of support for something would stop AMP from being widely used and deployed."
+msgstr ""
+
+msgid "No whitelists."
+msgstr ""
+
+msgid "We won’t give any special treatment to specific sites, domains or origins except where needed for security or performance reasons."
+msgstr ""
+
+msgid "Discover more..."
+msgstr ""
+
+msgid "Create your first AMP page"
+msgstr ""
+
+msgid "Not sure how to get started? In this tutorial, you’ll learn how to create a basic AMP HTML page, how to stage it and validate that it’s AMP compliant, and finally ..."
+msgstr ""
+
+msgid "In this tutorial, you'll learn how to use data binding and expressions to build awesome, interactive AMP pages with amp-bind ..."
+msgstr ""
+
+msgid "Styling and layout on AMP HTML pages is very similar to normal HTML pages – in both cases, you'll use CSS."
+msgstr ""
+
+msgid "Get Started with Tutorials"
+msgstr ""
+
+msgid "AMP Websites"
+msgstr ""
+
+msgid "AMP Email"
+msgstr ""
+
+msgid "advertisers"
+msgstr ""
+
+msgid "publishers"
+msgstr ""
+
+msgid "e-commerce"
+msgstr ""
+
+msgid "nonprofit"
+msgstr ""
+
+msgid "social-media"
+msgstr ""
+
+msgid "Magazine article"
+msgstr ""
+
+msgid "Breaking the status quo"
+msgstr ""
+
+msgid "This example of magazine article demonstrates how to build an immersive reading experience by adding animations that enhance the written content on the page."
+msgstr ""
+
+msgid "<code>amp-position-observer</code> in combination with <code>amp-animation</code> allows a smooth \"scrolly-telling\" experience with minimal effort and without any custom JavaScript."
+msgstr ""
+
+msgid "Experience Demo"
+msgstr ""
+
+msgid "Band page"
+msgstr ""
+
+msgid "Olga and the kings"
+msgstr ""
+
+msgid "Always thought that all AMP pages look the same? Checkout this incredible band website featuring various AMP components to create a vivid and enjoyable experience."
+msgstr ""
+
+msgid "The use of <code>amp-sidebar</code> gives you the same feeling as if you are in a single page app."
+msgstr ""
+
+msgid "E-Commerce"
+msgstr ""
+
+msgid "AMP Camp"
+msgstr ""
+
+msgid "AMP shines for content pages. But it can also power e-commerce experiences."
+msgstr ""
+
+msgid "To learn more, we built a simple yet fully functional e-commerce site of our very own! Check out the site, and look at the code to learn about the best practices we discovered to build a great interactive experience on your domain and on AMP caches alike."
+msgstr ""
+
+msgid "See on GitHub"
+msgstr ""
+
+msgid "Here we feature interesting and outstanding pages that have been published using AMP. We kindly invite you to look through them to see what AMP is capable of."
+msgstr ""
+
+msgid "Musement’s mobile-first design helps users navigate 70% faster"
```

Comment:
do we need to extract the brand name here? (same for the others below)

Reply:
I assumed the translation would just be their trademark name in the translation. I will add a translator instructions above it to make it clear that "Musement" (and other trade names) are left as is